### PR TITLE
middleware register

### PR DESCRIPTION
**Pull request by shubhamraj82**
## Description

**Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.**

Fixes # (issue) *(only if applicable)*

### Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other (please specify):

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### Screenshots (if applicable)

**Please include any relevant screenshots or images that help explain the changes made.**

#### Additional Information 
- Name: Shubham raj
- Email: 2229177@kiit.ac.in
- Roll No:2229177
- Branch: CSCE

### CODE REVIEW

File: middlewares/auth..js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const jwt = require("jsonwebtoken");
+export const verifyToken = async (req, res, next) => {
```
Comment (dead8309) [inline middlewares/auth..js:2]: Can you rename this to `authMiddleware`

**Comment by shubhamraj82** [comment]
done

**Comment by dead8309** [comment]
Great! Can you also mention which issue you are fixing?

**Comment by shubhamraj82** [comment]
https://github.com/MLSAKIIT/devblogs-backend/issues/17                 https://github.com/MLSAKIIT/devblogs-backend/issues/9
https://github.com/MLSAKIIT/devblogs-backend/issues/6                   https://github.com/MLSAKIIT/devblogs-backend/issues/4        https://github.com/MLSAKIIT/devblogs-backend/issues/3

**Comment by dead8309** [comment]
Thank you @shubhamraj82 for contributing to the project!

Unfortunately, I cannot accept your current pull request because it addresses multiple issues, and some of these issues are already assigned to other contributors.

Here’s how you can proceed:

- Please make sure an issue is assigned to you before you start working on it. You can request assignment by commenting on the issue you'd like to work on.
- Submit one PR per issue so we can manage and review your contributions more effectively.

Since some of the issues you worked on are not yet assigned. If you're interested, I can assign those to you so you can submit them separately as individual PRs. Feel free to check other open issues that are not yet assigned, or if you'd like, you can suggest new ones!

**Pull request closed without merging**